tests/phpunit/CRM/Activity/Form/Task/PDFTest.php

```diff
@@ -94,6 +94,8 @@ protected function getActivityTokens(): array {
       '{activity.source_contact_id}' => 'Source Contact',
       '{activity.target_contact_id}' => 'Target Contacts',
       '{activity.assignee_contact_id}' => 'Assignee Contacts',
+      '{activity._depth}' => 'Depth',
+      '{activity._descendents}' => 'Descendents',
```

Comment:
It's weird that [these fields](https://github.com/civicrm/civicrm-core/blob/b08b73564ccbf46dd3d0e5add2f7f25fb590bb32/Civi/Api4/Service/Spec/Provider/HierarchicalEntitySpecProvider.php#L29-L51) get picked up as tokens. I tried changing the `usage` flag of the fields but it didn't help. They already don't have `token` in their `usage` array. So is tokenProcessor ignoring `usage` @eileenmcnaughton ?